examples/web-component/webpack.config.js

```diff
@@ -0,0 +1,25 @@
+var HtmlWebpackPlugin = require('../..');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var webpackMajorVersion = require('webpack/package.json').version.split('.')[0];
+module.exports = {
+  entry: './example.js',
+  output: {
+    path: __dirname + '/dist/webpack-' + webpackMajorVersion,
+    publicPath: '',
+    filename: 'bundle.js'
+  },
+  module: {
+    loaders: [
+      { test: /\.css$/, loader: ExtractTextPlugin.extract('style-loader', 'css-loader') }
+    ]
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      inject: false,
+      cache: false,
```

Comment:
There actually no need for that.  I just copied and pasted from one of the examples.  Sorry for the sloppiness.  No need to disable cache.